android/modules/contacts/src/java/ti/modules/titanium/contacts/ContactsApiLevel5.java

```diff
@@ -286,8 +286,6 @@ protected int processIMProtocol(String serviceName)
 			return Im.PROTOCOL_QQ;
 		} else if (serviceName.equals("Skype")) {
 			return Im.PROTOCOL_SKYPE;
-		} else if (serviceName.equals("Yahoo")) {
```

Comment:
Should this be removed? Does Yahoo still have IM? I don't think it's related to our Ti.Yahoo API.